package.json

```diff
@@ -65,6 +65,8 @@
     "url": "https://github.com/mongodb/node-mongodb-native/issues"
   },
   "scripts": {
+    "tool:spec-update": "sh ./etc/update-spec-tests.sh",
+    "tool:cluster": "sh ./test/tools/cluster.sh",
```

Comment:
Since we're using bash specific syntax we can just launch with bash here

Reply:
done